### PR TITLE
Fix workflow testing YAML

### PR DESCRIPTION
**Pull request by erasche**
Started trying to run the workflow tests against usegalaxy.eu only to find many contained yaml issues that caused immediate test failures.

### CODE REVIEW

**Comment by shiltemann** [comment]
we could add the following to `.yamllint` to change the missing document start from warning to error?

```
document-start:
    level: error
    present: yes
```

**Comment by bgruening** [comment]
@shiltemann I can be mistaken, but I think someone ( @bebatut ? ) removed the `---` intentionally a few weeks ago from the linter.

**Comment by erasche** [comment]
@bgruening I don't see anything like that in the history https://github.com/galaxyproject/training-material/commits/master/.yamllint

File: Makefile [hunk 1]
```diff
@@ -115,7 +115,8 @@ check-slides: build  ## check the markdown-formatted links in slides
 
 check-yaml: ## lint yaml files
 	( $(ACTIVATE_ENV) && \
-	  find . -path "**/*.yaml" | xargs -L 1 -I '{}' sh -c "yamllint {}" \
+	  find . -name "*.yaml" | xargs -L 1 -I '{}' sh -c "yamllint {}" \
```
Comment (nsoranzo) [inline Makefile]: I thinkg we should also use single quotes for `*.yaml` to prevent shell expansion, i.e. `find . -name '*.yaml' | xargs ...`

Comment (erasche) [reply Makefile]: I don't believe expansion happens when double or single quoted but I could be wrong? I was only under the impression that variables are expanded, not file paths.

**Comment by shiltemann** [comment]
@bgruening yes, I relaxed some of the yaml rules a while ago (tho not specifically this one, it was always a warning) but if it gets in the way of your automatic workflow testing I thought it would be nice to require it? instead of you having to keep manually fixing it?

File: Makefile (hunk 1 above)
Comment (shiltemann) [reply Makefile]: ok, will do @nsoranzo :)

Comment (nsoranzo) [reply Makefile]: Ah, @erasche is right, sometimes Bash is so confusing!

**Pull request merged**